### DISCUSSION TITLE
Blogging on Github Enterprise

### DISCUSSION THREAD

**Discussion by SilentGlasses** [community/community#37586] [Enterprise]
I was wondering if anyone has been able to build a blog on Github Enterprise Pages.  I love the [Chirpy theme](https://chirpy.cotes.page/) but not sure if this is possible on an enterprise setup.

**Comment by ossanna16** [comment]
Hi there @SilentGlasses and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jfullstackdev** [comment]
fortunately, there is the one developed that on top of Jekyll 

https://github.com/cotes2020/jekyll-theme-chirpy

you can even ask them there

since Jekyll is the static generator of GitHub Pages, enjoy the page theme

**Reply by SilentGlasses** [reply]
I use that on GitHub but it does not work on Enterprise, I did also ask there and the creator sent me here. :( 
https://github.com/cotes2020/jekyll-theme-chirpy/issues/737
